config/initializers/sidekiq.rb

```diff
@@ -1,7 +1,7 @@
 Sidekiq.configure_server do |config|
-  config.redis = { :url => 'redis://localhost:6379', :namespace => 'sidekiq' }
+  config.redis = { :url => 'redis://localhost:6379' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.